.travis.yml

```diff
@@ -34,6 +33,9 @@ jobs:
       name: "Run tests for Scala 2.12 and Java 11"
     - script: bin/test-documentation
       name: "Documentation validations and tests"
+    - script: bin/run-docs-tests
```

Comment:
👨‍🎨 I don't like the name, because it breaks from `bin/test-<x>`.

Perhaps `bin/test-docs-only`, or something like that?